src/main/java/io/confluent/connect/hdfs/FileUtils.java

```diff
@@ -101,7 +102,12 @@ public static String committedFileName(
   }
 
   public static String topicDirectory(String url, String topicsDir, String topic) {
-    return url + "/" + topicsDir + "/" + topic;
+    boolean topicInPath = Boolean.valueOf(StorageCommonConfig.PATH_INCLUDE_TOPICNAME_CONFIG);
```

Comment:
This will always be false since you're parsing the config string, not looking up the value in the config